### DISCUSSION TITLE
Expand from a single developer to a ninja clan?

### DISCUSSION THREAD

**Discussion by SmileyChris** [vitalik/django-ninja#396] [General]
Hey @vitalik -- what are your thoughts about project maintenance long term. Are you happy to just retain sole ownership of the repo or is it something you'd be interested in sharing the workload?

Personally, I'd be more than happy to join a team with permission to commit and keep things moving, while still working under your guidance as to the project's direction.

**Comment by vitalik** [comment]
Hi @SmileyChris 

indeed support and progress is getting a bit overwhelming  ( + [the war issue](https://github.com/vitalik/django-ninja/issues/383) ) 

I will try to outline the next goals soon 
Just drop me a "hi" to ppr.vitaly@gmail.com 

Cheers
Vitaliy

**Reply by magedhelmy1** [reply]
Hi @vitalik ! I hope you are well. What is the update with regards to this issue? Is there possibility to create "contributing" section in the repo. Thanks in advance.

**Reply by OtherBarry** [reply]
@vitalik did this go anywhere?

I feel llke having a few people with the ability to close issues, merge pull requests, and generally maintain the repository would really improve the project. It lets you spend your time on the more important stuff, and keeps the repository more active and friendly to new users.

**Reply by vitalik** [reply]
@OtherBarry 

I'm currently allocating some planning - and I think the most important and breaking will be a migration to upcoming pydantic2
Most likely it will be incompatible version that will lead to django-ninja major version upgrade (to 1.xx.xx) but on the other hand it will have some long awaited features

**Reply by fretchen** [reply]
From your answers it would seem that you have limited interested in expanding towards a broader maintainer base ? Is this correct ? How would you like to keep it managable then ?

**Comment by baseplate-admin** [comment]
Hi,

I think adding some people to traige issues for this repo would be great as `django-ninja` has over 150 open issues and it shows a lack of maintainance

**Comment by rednaks** [comment]
Would be happy to help as well.

**Comment by jkeyes** [comment]
I'd be happy to contribute too.

**Comment by vitalik** [comment]
Well things need some collective brainsorm I guess :)

as of upcoming pydantic v2 - the GetterDict was removed (while it was very essential part of Schemas) 

Ideally need to find solution that should work just fine in pydantic v2 or v1 without a need for django-users to think which version to use

Details here https://github.com/pydantic/pydantic/issues/5440 - feel free to add there any ideas or hacks if you can think of any

cc @SmileyChris @jkeyes @rednaks @baseplate-admin

**Reply by SmileyChris** [reply]
Splitting the discussion for brainstorming getterdict to #738 

I think the main push in the discussion of this thread @vitalik is to ensure the long-running welbeing of ninja as a maintained project. Moving it to a custom github org, and giving some trusted users some discretion for basic bug fixing and having PR reviewers you can trust would be a good start.

I'm sure this could be done in a way that still ensuring your vision for the project is met, and high standards for code quality and minimal bloat be maintained. Having a larger maintainer pool can bring is more trust to the project with and hopefully increase the filtering of new ideas, potentially even optimising the development cycle.

**Reply by baseplate-admin** [reply]
Specially since `fastapi` had a single maintainer and it caused the community to make [`litestar`](https://github.com/litestar-org/litestar) and ninja now has competition in form of [`django-hatchway`](https://pypi.org/project/django-hatchway/)

<sup>

Relevant Links:

* [github discussion/issue](https://github.com/tiangolo/fastapi/issues/4263) 


* [Hackernews](https://news.ycombinator.com/item?id=27610383)

</sup>

**Comment by vitalik** [comment]
Hi

I'm starting marking tickets that are good candidates for next release with "help wanted" tag - PRs are welcome

https://github.com/vitalik/django-ninja/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22
